### DISCUSSION TITLE
Multiple instances of MINIO on one server

### DISCUSSION THREAD

**Discussion by partygailo** [minio/minio#20003] [Q&A]
Hi there. 
I try to run multiple instances of minio on one server. 
I'm running them on different IP's, ports and binary files. I created different systemd services for them.  Here is the config: 
_prod_ 
/usr/local/bin/minio_pr server --certs-dir /home/minio-user/.minio/certs --console-address 10.16.0.142:9001 /srv/prod
_test_
/usr/local/bin/minio server --certs-dir /home/minio-user/.minio/certs --console-address 10.16.0.202:9002 /srv/test

I also use "MINIO_ADDRESS=" option to run instances with HTTPS protocol:
MINIO_ADDRESS="prod.mydomain.com:443
MINIO_ADDRESS="test.mydomain.com:443

When I started both of them, I saw a very interesting thing while using netstat command:
prod and test instances are running on the IP's from the configuration by 443 port, but i also see a 443 port running on the localhost.  
_tcp        0      0 127.0.0.1:443           0.0.0.0:*               LISTEN      138048/minio        
tcp        0      0 10.16.0.142:9001        0.0.0.0:*               LISTEN      137151/minio_pr     
tcp        0      0 10.16.0.202:9002        0.0.0.0:*               LISTEN      138048/minio        
tcp        0      0 10.16.0.142:443         0.0.0.0:*               LISTEN      137151/minio_pr     
tcp        0      0 10.16.0.202:443         0.0.0.0:*               LISTEN      138048/minio_        

I read the [thread](https://github.com/minio/minio/discussions/17817) which says that the instance always binds to localhost, and it's clear with one instance of minio. But in my situation it's became more difficult. 

When I start to use browser to manage minio by different "MINIO_ADDRESS": 
 prod.mydomain.com:443 or test.mydomain.com:443, all my requests are going to the instance, which is binding on the localhost.  

First question  - is such configuration are normal and can be used?
Second - how i can change this behavior of minio? 
Third - how to manage this instances by web?

**Comment by marktheunissen** [comment]
Hi @partygailo, why are you trying to run MinIO like this? What are your objectives? Why not follow the docs?

This is not a recommended configuration.

**Reply by partygailo** [reply]
Hi, @marktheunissen. I want to deploy the Minio in such way because I don't want to deploy another virtual server for low loaded test environment.  In previous deployment of Minio in our company, my colleagues install Minio in the way that i describe.

**Reply by marktheunissen** [reply]
If you want a single server then take a look at the docs here: https://min.io/docs/minio/linux/operations/install-deploy-manage/deploy-minio-single-node-multi-drive.html#minio-snmd

There is just one MinIO process needed.

**Comment by manju-rn** [comment]
Run minio in kuberentes using the operator. You can create multiple tenants
